src/views/Dashboard.js

```diff
@@ -86,12 +86,25 @@ const useStyles = makeStyles((theme) => ({
   buttonContainer: {
```

Comment:
basically what should be done is to create our own buttonGroup component with the appropriate styling and use that component in all 3 places. That is also part of the refactoring task, to create components for these new styles and not duplicate styling code in all the different places where it is used.

It is also used in MyProfile.js where it is also currently broken

Reply:
I refactor it a bit more